packages/sources/vue/src/components/indicators/VtmnPrice/VtmnPrice.vue

```diff
@@ -0,0 +1,41 @@
+<script lang="ts">
+import '@vtmn/css-price/dist/index-with-vars.css';
+import { reactive, computed, defineComponent, PropType } from 'vue';
+import { VtmnPriceSize, VtmnPriceVariant } from './types';
+
+export default /*#__PURE__*/ defineComponent({
+  name: 'VtmnPrice',
+  props: {
+    variant: {
+      type: String as PropType<VtmnPriceVariant>,
+      default: 'primary',
+      validator: (val: VtmnPriceVariant) =>
+        ['default', 'accent', 'alert', 'strikethrough'].includes(val),
+    },
+    size: {
+      type: String as PropType<VtmnPriceSize>,
+      default: 'medium',
+      validator: (val: VtmnPriceSize) =>
+        ['small', 'medium', 'large'].includes(val),
+    },
+  },
+  setup(props) {
+    props = reactive(props);
+    return {
+      styleObject: {
+        color: 'inherit',
+        fontSize: 'inherit',
+      },
+      classes: computed(() => ({
+        'vtmn-price': true,
+        [`vtmn-price_variant--${props.variant}`]: true,
```

Comment:
```suggestion
        [`vtmn-price_variant--${props.variant}`]: props.variant,
```

Isn't it better to display the class of the variant only if a value is set ? (same for the size)